.circleci/config.yml

```diff
@@ -98,6 +98,21 @@ executors:
       - image: python:<<parameters.python_version_major>>.<<parameters.python_version_minor>>
     resource_class: xlarge
 
+  local-testcontainer:
```

Comment:
somewhat confusingly, we also have a job called "local-testcontainer", but to be honest i'm not sure what it does. maybe it would help to rename that one, if you know what it does

Reply:
i think this on is used to store the released test containers in our own registry. yeah, i was looking this one as well... i can give it a more descriptive name